.github/_support/github-slack-notify.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+"""
+A script which runs as part of our GitHub Actions build to send notifications
+to Slack. Notifies when build status changes.
+
+Usage:
+    ./github-slack-notify.py $STATUS
+
+where $STATUS is the current build status.
+Requires GITHUB_TOKEN and SLACK_WEBHOOK_URL to be in the environment.
+"""
+import os
+import sys
+
+import requests
+
+
+def statusemoji(status):
+    return {"success": ":heavy_check_mark:", "failure": ":rotating_light:"}.get(
```

Comment:
Very tempting to suggest this:

```diff
-    return {"success": ":heavy_check_mark:", "failure": ":rotating_light:"}.get(
+    return {"success": ":heavy_check_mark:", "failure": ":kingkong:"}.get(
```

But I think maybe the warning light is easier for new folks to understand.